radixdlt-java/src/main/java/com/radixdlt/client/core/network/actions/FindANodeResultAction.java

```diff
@@ -0,0 +1,43 @@
+package com.radixdlt.client.core.network.actions;
+
+import com.radixdlt.client.core.network.RadixNode;
+import com.radixdlt.client.core.network.RadixNodeAction;
+import java.util.Objects;
+
+/**
+ * The result of a FindANodeRequestAction.
```

Comment:
Should possibly be "{@link FindANodeRequestAction}" to make it easier for readers.